README.md

```diff
@@ -42,6 +42,11 @@
 
 - [Beancount.io](https://beancount.io/) - Out-of-box Beancount and Fava SaaS that you can use instantly
 
+## Commercial Apps
+
+- [Beancount.io iOS App](https://apps.apple.com/us/app/id1527950512) - First-ever App on iOS that visualizes and connects to your Beancount ledger elegantly
+- [Beancount.io Android App](https://play.google.com/store/apps/details?id=io.beancount.android) - Add a transaction to your Beancount ledger quickly
```

Comment:
Is it possible to reframe the descriptions here to something neutral?

"First-ever App on iOS that visualizes and connects to your Beancount ledger elegantly" sounds a bit advertise-y and ideally I'd like to keep everything on awesome-beancount.com neutral.

One suggestion would be "iOS app for Beancount.io", and the same for the Android app. What do you think?